hardhat.config.ts

```diff
@@ -12,8 +12,8 @@ import * as dotenv from 'dotenv';
 
 dotenv.config();
 
-const secret: string = process.env.MNEMONIC_OR_PRIVATE_KEY as string;
-const etherscanKey: string = process.env.ETHERSCAN_API_KEY as string;
+// const secret: string = process.env.MNEMONIC_OR_PRIVATE_KEY as string;
+// const etherscanKey: string = process.env.ETHERSCAN_API_KEY as string;
```

Comment:
Don't comment out this line.

Reply:
The etherscanKey variable was not used anywhere, so it did not pass  lint checks. 

To pass the check, uncomment code:
```
etherscan: {
    // Your API key for Etherscan
    // Obtain one at https://etherscan.io/
    apiKey: etherscanKey,
}
```